ui-v2/app/adapters/node.js

```diff
@@ -1,6 +1,14 @@
 import Adapter from './application';
 import { PRIMARY_KEY, SLUG_KEY } from 'consul-ui/models/node';
 import { OK as HTTP_OK } from 'consul-ui/utils/http/status';
+// TODO: Looks like ID just isn't used at all
```

Comment:
What do you mean that ID isn't used at all here? It looks like you're generating IDs on the client and using that as the primary key (`uid`) so ember data is never using the ID from the server for its identity map. Probably not an issue as long as you're not caching the list or editing things, but something to be aware of if you ever want to add that sort of thing. I'm guessing you already work around that though for individual node pages (or you fetch the group first).

Reply:
Hey, yeah its more or less a note to go back and consider just ignoring `ID` completely, and just pretend `Node` is the 'unique key' to use for the slug. It sounds like that is always unique and `ID` can sometimes be empty, although, even while I'm writing this, the more i think about it the more I'd just continue to use `ID` - it just reads better plus the big has been fixed in the backend now - this is just for old data I think.